jll/A/autoconf_jll/Compat.toml

```diff
@@ -1,10 +1,11 @@
 [2]
 julia = "1.6.0-1"
 
-["2-2.71"]
+["2 - 2.71"]
 JLLWrappers = "1.2.0-1"
 
-["2.72-2"]
+["2.72 - 2"]
 Artifacts = ["0.0.0", "1"]
 JLLWrappers = "1.7.0-1"
 Libdl = ["0.0.0", "1"]
+julia = "1.6.0 - 1"
```

Comment:
```suggestion
```
Argh where did that come from? 

@giordano can we just drop it? Probably not and the wrapper needs it ... argh 

But I still don't understand: looking at https://github.com/JuliaBinaryWrappers/autoconf_jll.jl/commit/51a7f8a77f3a9e30dc649265653e01a3bf7cb29a it seems this compat requirement was there before already, just not reflected in the registry?!

Reply:
I suspect this is a bug in RegistryTools.

Reply:
Hrm, this looks similar: https://github.com/JuliaRegistries/General/pull/143667/files?diff=unified&w=1

And then @eschnett reported https://github.com/JuliaRegistries/RegistryTools.jl/issues/108 for https://github.com/JuliaRegistries/General/pull/143468

Reply:
OK based on what I gather from the issue and proposed fix, the patch I propose above seems to actually be a correct workaround for the RegistryTools bug, as the range `julia = "1.6.0-1"` is already present in line 2, and RegistryTool simply mistakenly considers this to be different from `julia = "1.6.0 - 1" .